app/lib/theme/widgets/SelectParentWidget.tsx

```diff
@@ -0,0 +1,79 @@
+import { WidgetProps } from "@rjsf/core";
+import Dropdown from "@button-inc/bcgov-theme/Dropdown";
+import getRequiredLabel from "../utils/getRequiredLabel";
+import { useState } from "react";
+
+const SelectParentWidget: React.FunctionComponent<WidgetProps> = (props) => {
```

Comment:
You could create an interface that extends `WidgetProps` so you can have static typing on `parent`, `child` and `foreignKey` 

```typescript
interface Props extends WidgetProps {
  parent: EntitySchema,
  child: EntitySchema,
  foreignKey: string
}
```